src/main/java/nl/tudelft/dnainator/core/impl/SequenceNodeImpl.java

```diff
@@ -20,29 +22,29 @@
 	/**
 	 * Constructs a default sequence with all parameters specified.
 	 * @param id The ID of this sequence.
-	 * @param source The sources of the sequence (from where it was sequenced).
+	 * @param sources The sources of the sequence (from where it was sequenced).
 	 * @param start The start position of the sequence.
 	 * @param end The end position of the sequence.
 	 * @param sequence The sequence.
 	 */
-	public SequenceNodeImpl(String id, String source, int start, int end, String sequence) {
-		this(id, source, start, end, sequence, 0, new ArrayList<>());
+	public SequenceNodeImpl(String id, List<String> sources, int start, int end, String sequence) {
+		this(id, sources, start, end, sequence, 0, new ArrayList<>());
 	}
 
 	/**
 	 * Constructs a default sequence with all parameters specified.
 	 * @param id The ID of this sequence.
-	 * @param source The sources of the sequence (from where it was sequenced).
+	 * @param sources The sources of the sequence (from where it was sequenced).
 	 * @param start The start position of the sequence.
 	 * @param end The end position of the sequence.
 	 * @param sequence The sequence.
 	 * @param rank The rank.
 	 * @param outgoing The neighbours
 	 */
-	public SequenceNodeImpl(String id, String source,
+	public SequenceNodeImpl(String id, List<String> sources,
```

Comment:
Might be more future proof to add a constructor which takes a Set.

The `SequenceNodeFactoryImpl` currently splits on commas, returning a List.